scalding-core/src/main/scala/com/twitter/scalding/typed/TypedPipe.scala

```diff
@@ -44,7 +44,20 @@ object TypedPipe extends Serializable {
    */
   def from[T](pipe: Pipe, fields: Fields)(implicit flowDef: FlowDef, mode: Mode, conv: TupleConverter[T]): TypedPipe[T] = {
```

Comment:
:+1: to postponing the decision on how we're going deal with the cascading dependency. I think it's a separate discussion.